modules/rosa-sts-creating-cluster.adoc

```diff
@@ -14,13 +21,25 @@ link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html[AWS Share
 
 .Procedure
 
-When creating your cluster, use the following default options to configure your networking IP ranges. For more information when using manual mode, use `rosa create cluster --help | grep cidr`. In interactive mode, you are prompted for the settings.
-
-* Machine CIDR: 10.0.0.0/16
-* Service CIDR: 172.30.0.0/16
-* Pod CIDR: 10.128.0.0/14
+. Create the required account-wide roles and policies, including Operator policies:
++
+[source,terminal]
+----
+$ rosa create account-roles --version 4.8 <1>
+----
+<1> You must specify an OpenShift release by using the `--version` option.
++
+You can select from the following modes in the interactive prompt:
++
+* `auto`: The account roles and and policies are created directly using the current AWS account.
+* `manual`: The role and policy JSON files are saved in the current directory. The command output includes the `aws` CLI commands required to create the roles.
```

Comment:
Should we clarify the aws-cli version that used for the manual mode commands execution? I used 2.0.x aws-cli before which cannot create the policy with option --tags in the output command. Then it worked after I upgrade the aws-cli to 2.4.x

Reply:
Thank you for highlighting this. Do you know if 2.4 is the minimum required version for this functionality?

Reply:
I have reviewed the `aws-cli` changelog and cannot see a reference to this. As discussed, I have subsequently updated the prerequisite to "* You have installed and configured the latest AWS, ROSA, and `oc` CLIs on your installation host."